File: executable/julia/text-chart-basic.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "x" => [0, 1, 2], 
    "y" => [1, 1, 1], 
    "name" => "Lines, Markers and Text", 
    "text" => ["Text A", "Text B", "Text C"], 
    "type" => "scatter", 
    "mode" => "lines+markers+text", 
    "textposition" => "top"
  ], 
  [
    "x" => [0, 1, 2], 
    "y" => [2, 2, 2], 
    "name" => "Markers and Text", 
    "text" => ["Text D", "Text E", "Text F"], 
    "type" => "scatter", 
    "mode" => "markers+text", 
    "textposition" => "bottom"
  ], 
  [
    "x" => [0, 1, 2], 
    "y" => [3, 3, 3], 
    "name" => "Lines and Text", 
    "text" => ["Text G", "Text H", "Text I"], 
    "type" => "scatter", 
    "mode" => "lines+text", 
    "textposition" => "bottom"
  ]
]

layout = ["showlegend" => false]

response = Plotly.plot([data], ["layout" => layout, "filename" => "text-chart-basic", "fileopt" => "overwrite"])
plot_url = response["url"]

File: executable/node/text-chart-basic.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var trace1 = {
  x: [0, 1, 2], 
  y: [1, 1, 1], 
  text: ["Text A", "Text B", "Text C"], 
  textposition: "top", 
  name: "Lines, Markers and Text", 
  mode: "lines+markers+text", 
  type: "scatter"
};
var trace2 = {
  x: [0, 1, 2], 
  y: [2, 2, 2], 
  text: ["Text D", "Text E", "Text F"], 
  textposition: "bottom", 
  name: "Markers and Text", 
  mode: "markers+text", 
  type: "scatter"
};
var trace3 = {
  x: [0, 1, 2], 
  y: [3, 3, 3], 
  text: ["Text G", "Text H", "Text I"], 
  textposition: "bottom", 
  name: "Lines and Text", 
  mode: "lines+text", 
  type: "scatter"
};
var data = [trace1, trace2, trace3];
var layout = {showlegend: false};

var graph_options = {filename: "text-chart-basic", fileopt: "overwrite", layout: layout}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});